src/main/java/traffic/TrafficLightColour.java

```diff
@@ -0,0 +1,15 @@
+package main.java.traffic;
+
+//An enumeration of the colors of a traffic light. 
+enum TrafficLightColour {
+	RED(1000), RED_YELLOW(500), GREEN(1000), YELLOW(1000);
```

Comment:
Ah, that's fair enough then - didn't realise it was the intermediary state. :+1: